dandi/consts.py

```diff
@@ -100,6 +102,7 @@ class DandiInstance:
     name: str
     gui: str | None
     api: str
+    is_private: Optional[bool] = None
```

Comment:
Why not have this default to `False` and get rid of the `Optional`?

Where is this attribute even used?

Reply:
Removed -- per https://github.com/dandi/dandi-cli/pull/1519#issuecomment-2471571634